### PR TITLE
Testing Jenkins setup

### PR DESCRIPTION
**Pull request by rameshthoomu**
Testing Jenkins build skip functionality
## Description

Testing Jenkins build skip functionality
## Motivation and Context

Fixes #
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [] Either no new documentation is required by this change, OR I added new documentation
- [] Either no new tests are required by this change, OR I added new tests
- [] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by: thoomu@us.ibm.com

### CODE REVIEW

**Comment by rameshthoomu** [comment]
test message

**Comment by rameshthoomu** [comment]
reverify jenkins

**Comment by rameshthoomu** [comment]
reverify jenkins

**Comment by rameshthoomu** [comment]
travis_retry

**Comment by christo4ferris** [comment]
@rameshthoomu @bramwelt we have a number of these faux PRs that all seem to be serving a similar if not identical purpose. Can we consolidate so that we don't unnecessarily inflate the number of open PRs? Thanks

**Comment by rameshthoomu** [comment]
@christo4ferris: I have created this PR to test ci-skip functionality in Jenkins at commit level. Closing this PR as we have PR#1715 and PR#1716 opened for Jenkins testing.

**Comment by rameshthoomu** [comment]
reverify Jenkins

File: core/ledger/ledgermgmt/ledgermgmttest/ledgermgmttest.go [hunk 1]
```diff
@@ -50,3 +57,34 @@ func NewInitializer(testLedgerDir string) *ledgermgmt.Initializer {
 		ChaincodeLifecycleEventProvider: &mock.ChaincodeLifecycleEventProvider{},
 	}
 }
+
+// CreateSnapshot creates a snapshot with only genesis block for the given ledgerID and returns
+// the snapshot directory. It is intended to be used for creating a ledger by snapshot for testing purpose.
+// The testDir parameter should be different from the ledger root dir used to import the snapshot;
+// otherwise, it will cause "ledger already exists" error when creating ledger from the snapshot.
+func CreateSnapshot(t *testing.T, testDir string, ledgerID string, configTxProcessor ledger.CustomTxProcessor) string {
```
Comment (manish-sethi) [inline core/ledger/ledgermgmt/ledgermgmttest/ledgermgmttest.go]: - Better to be explicit in function name `CreateSnapshotWithGenesisBlock` (or take the list of blocks from outside)
- The side effect of the creation of ledgerMgr instance should be avoided. Better not to take the `testDir` as input. Instead, create a tempdir and delete before returning from this function.

File: internal/peer/channel/channel.go [hunk 2]
```diff
@@ -76,6 +80,7 @@ func resetFlags() {
 	flags = &pflag.FlagSet{}
 
 	flags.StringVarP(&genesisBlockPath, "blockpath", "b", common.UndefinedParamValue, "Path to file containing genesis block")
+	flags.StringVarP(&snapshotPath, "snapshotPath", "", common.UndefinedParamValue, "Path to the snapshot directory")
```
Comment (manish-sethi) [inline internal/peer/channel/channel.go]: `snapshotPath` -> `snapshotpath` just to look similar to `blockpath`

**Pull request closed without merging**